slackbridge/config.json.sample

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "nickname": "test",
+    "server": "irc.bottest.org",
+    "token": "slacktoken",
+    "autoSendCommands": [
+        ["MODE", "test", "+i"],
+        ["AUTH", "test", "password"]
+    ],
+    "channelMapping": {
+      "#slack": "#irc",
+      "privategroup": "#other-channel"
+    }
+  },
+  {
+    "nickname": "test2",
+    "server": "irc.testbot.org",
+    "token": "slacktoken2",
+    "channelMapping": {
+      "#other-slack": "#new-irc-channel"
+    }
+  }
+]
```

Comment:
it looks like you ended up including the real json *and* the sample. i only want the sample.

then you can update your docker run example to include the var's inline. that makes it easier for users...and honestly, you could contribute this change back to the author (if they want it).